### DISCUSSION TITLE
Cats and Dogs

### DISCUSSION THREAD

**Discussion by bm55** [blakeblackshear/frigate#391] [General]
@blakeblackshear any thoughts on merging cats and dogs to one label? 
![image](https://user-images.githubusercontent.com/7528001/102503950-1bdf5e80-4035-11eb-94bf-a7994a94a39e.png)

**Comment by tubalainen** [comment]
Hi @bm55!

AI detection of any object from "above" is very hard, hence the distinguishing features of each trained object is based on the characteristics of that object, sure, that includes being able to detect "a cat" from above. A small dog and a cat, seen from above can fool even a human eye if the dog and the cat has the same size.

I am quite surprised to see that it did such a great job as it did above.

Detecting stuff from "the side perspective" rather than from "the above perspective" provides much more attributes to base the model and model category on.

I have a problem here at home that my robotic lawn mower sometimes been detected as a person. I adjusted the size of an person and the issue went away - this is collaboration with zones and the filters per object. I have made some presumptions, for instance, a human does not fly or hangs from the ledge of my house (mask in this case).
Perhaps you can tweak the box sizes of each object type to tweak this?
I guess moving the camera to another location to provide a more "from the side view" is not an option?

**Comment by mattheys** [comment]
> @blakeblackshear any thoughts on merging cats and dogs to one label?

You can already do this by customising the labelmap.  https://github.com/blakeblackshear/frigate/blob/893e6b40a73474859aa6f2a797a06319aa0d6066/README.md#customizing-the-labelmap

**Reply by julianrinaldi** [reply]
Is there a way to do this easily with the Hassio add-on?

**Reply by iMiMx** [reply]
> Is there a way to do this easily with the Hassio add-on?

Did you find a solution to this?
